fabric-transitive-access-wideners-v1/src/main/resources/fabric-transitive-access-wideners-v1.accesswidener

```diff
@@ -113,6 +113,10 @@ transitive-accessible method net/minecraft/village/VillagerType <init> (Ljava/la
 # Access stack list in SimpleInventory
 transitive-accessible field net/minecraft/inventory/SimpleInventory stacks Lnet/minecraft/util/collection/DefaultedList;
 
+# Creating custom Sound event's
```

Comment:
```suggestion
# Creating custom sound events
```

Reply:
Good thinking, we should add something to check that there arent any generated file chages 🤔

Reply:
Seems simple enough, I could make a PR for a task to check that the real generated file matches the output